packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -159,9 +166,17 @@ contract Deploy is Deployer {
     /// @notice Deploy a new OP Chain using an existing SuperchainConfig and ProtocolVersions
     /// @param _superchainConfigProxy Address of the existing SuperchainConfig proxy
     /// @param _protocolVersionsProxy Address of the existing ProtocolVersions proxy
-    function runWithSuperchain(address payable _superchainConfigProxy, address payable _protocolVersionsProxy) public {
+    /// @param _sharedLockboxProxy Address of the existing SharedLockbox proxy
+    function runWithSuperchain(
+        address payable _superchainConfigProxy,
+        address payable _protocolVersionsProxy,
+        address payable _sharedLockboxProxy
+    )
+        public
```

Comment:
note: These are the 3 contracts that are cluster wide, where there is only a single contract per the entire cluster